README.md

```diff
@@ -1,8 +1,37 @@
-# epuck_ros2
+# E-Puck Driver for ROS2
 [![Build Status](https://travis-ci.com/cyberbotics/epuck_ros2.svg?branch=master)](https://travis-ci.com/cyberbotics/epuck_ros2)
 [![license - apache 2.0](https://img.shields.io/:license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Version](https://img.shields.io/github/v/tag/cyberbotics/epuck_ros2?label=version)](http://wiki.ros.org/epuck_ros2)
 
+<img src="./assets/pi-puck.jpg" width="400px" />
+
+This ROS2 package adds support for [e-puck](https://www.gctronic.com/doc/index.php/e-puck2) physical robot with [Pi-puck extension](https://www.gctronic.com/doc/index.php?title=Pi-puck).
+Please use the following links for the instructions:
+- [Installation](https://github.com/cyberbotics/epuck_ros2/tree/master/installation)
+- [Getting Started](https://github.com/cyberbotics/epuck_ros2#getting-started)
+- [Examples](https://github.com/cyberbotics/webots_ros2/blob/doc-epuck-refactor/webots_ros2_epuck/EPUCK_ROS2.md)
+- [Simulation](https://github.com/cyberbotics/webots_ros2/tree/doc-epuck-refactor/webots_ros2_epuck)
```

Comment:
Same for these links, in any case they will not work once the ``doc-epuck-refactor`` will be removed.